src/App.jsx

```diff
@@ -1,25 +1,35 @@
 import React from 'react'
 import { Route, Routes} from 'react-router-dom';
-
 import Header from './components/Header';
 import CardList from './components/CardList';
 import SingleView from './components/SingleView';
+import Cart from './components/Cart';
+import {CartProvider} from './state/CartProvider';
 import productData from './data/full-products';
 
 
 function App() {
-  
+
   return (
     <div className="App">
       <Header />
-      
+
         <Routes>
           <Route path="/" element={<CardList data={productData} />} />
           <Route path="/product/:id" element={<SingleView data={productData} />} />
         </Routes>
-      
+    <CartProvider>
+    <Header />
+
+    <Routes>
+      <Route path="/" element={<CardList />} />
+      <Route path="/product/:id" element={<SingleView />} />
+      <Route path="/cart" element={<Cart />} />
+      <Route path="/orders" element={<Orders />} /> 
+    </Routes>
```

Comment:
You're redeclaring the `Routes` here, they are already declared above on line 17.